### ISSUE TITLE
Set tag on conversation when Github Issue is linked

### ISSUE THREAD

**Issue by josephwegner** [josephwegner/carrick-issues-tracker#2]
Needs more customer research. Questions:

- Does the tag need to have the Issue ID in it? `github-issue` vs `github-issue-432`
- How should the tags be set if there are multiple issues on the same conversation? Just double them up?

**Comment by CLBarajas** [comment]
FWIW, I've previously been advised by Help Scout _not_ to add a lot of individual tags, because if you end up with a large number (which was not further specified) it can start to cause performance issues within Help Scout.

Up until we started using Carrick, we were applying issues with the tag format `github-123`, based on the issue number. (We'd previously tried an even shorter option of `gh#123` but, while the octothorp can be used in a tag, it's nearly impossible to search for within Help Scout.) Keep in mind also, if you wanted to track items across multiple repositories you'd need to use a tag format that included it, i.e. `github-support-123`.

I could see a system where you tag all linked conversations from Carrick itself in its all issues view. For example, when an issue is closed in GitHub, you could find the related conversations from Carrick and briefly apply a `follow-up` tag to make outgoing contact to the affected users. (So, maybe a custom tag option?)

**Comment by josephwegner** [comment]
@CLBarajas what's your use case for wanting to tag all of them? Is it just for the case of a future batch-follow-up message, like when the linked feature ships?

**Comment by CLBarajas** [comment]
Yeah, that's definitely the first idea there - you could set up an automatic workflow to send an automated reply to any item tagged `follow up`, and also delete the tag or apply a new tag of `contacted`.
